### ISSUE TITLE
Just a blank screen

### ISSUE THREAD

**Issue by hkjels** [dickmao/nnreddit#68]
From the readme, it sounds like the Oauth-process will start by itself once I open Gnus, but all I get is a blank screen. I have never used GNU's before; will I need any additional setup?

**Comment by dickmao** [comment]
> I have never used gnus before; will I need any additional setup?

Gnus alas requires elisp knowledge.  I don't know anyone, least of all myself, who hasn't spent as much time working through Gnus's peculiarities as they did learning emacs itself.

A number of moving parts like virtualenv, your python and emacs installations, and your browser setup play into the oauth handshake.  I've only ever tested it on Ubuntu linux.

As Gnus (and mail in general) is such a sprawling mess, the only reasonable way of troubleshooting is via `make test-run-interactive`.  You'd need to install [Cask](https://cask.readthedocs.io/en/latest/) and possess more than basic UNIX and elisp wherewithal.

In short, Gnus is not for everyone.  I'm happy to answer questions, but the struggle is real.
